src/Tickets/Admin/Onboarding/Controller.php

```diff
@@ -118,34 +118,38 @@ public function handle_onboarding_page_dismiss(): void {
 	 * @return void
 	 */
 	public function redirect_tec_pages_to_guided_setup(): void {
+		$force = apply_filters( 'tec_tickets_onboarding_force_redirect_to_guided_setup', false );
 		// Do not redirect if they are already on the Guided Setup page.
 		$page = tec_get_request_var( 'page' );
 		if ( Landing_Page::$slug === $page ) {
 			return;
 		}
 
-		// Do not redirect if they have been to the Guided Setup page already.
-		if ( (bool) tribe_get_option( Landing_Page::VISITED_GUIDED_SETUP_OPTION, false ) ) {
-			return;
-		}
+		if ( ! $force ) {
```

Comment:
This needs a check for if we are on a proper page to begin with. Otherwise the force will redirect the entire admin 🤦🏼‍♂️